### ISSUE TITLE
Favicon is not getting applied.

### ISSUE THREAD

**Issue by Anish-Karthik** [adrianhajdin/threads#14]
how to apply the favicon?
i have applied everything same as in video

**Comment by jaydeeprana049** [comment]
> how to apply the favicon? i have applied everything same as in video



> how to apply the favicon? i have applied everything same as in video

convet favicon from .svg to fevicon

**Comment by vinod1147** [comment]
> how to apply the favicon? i have applied everything same as in video

first, convert the logo.svg file to favicon.ico with the help of the favicon.io website after that download the icon and unzip the zip file then paste the icon into the public folder and delete the existing icon.

**Comment by rayvinith** [comment]
yes in the video the favicon part was i thing missed better try to see in internet you will get ,

**Comment by neophyte-programmer** [comment]
I had the same issue even though i followed the steps in the video so i researched and used this method. 

1. Place the favicon.ico in your `public/assets/` folder 
2. In the layout.tsx file, type this

```jsx
import { Metadata } from "next";

export const metadata: Metadata = {
    title: 'Echo: Speak your mind',
    description: 'Social media for the people',
    icons: { icon: '/assets/favicon.ico' },
  }
```

This worked for me.

**Comment by i1Cps** [comment]
@neophyte-programmer solution worked for me but also the favicon in general will not work in some browsers such as Brave due to shields.

**Comment by AymanBoumait** [comment]
the favicon won't get applied sometimes, unless you host the website

**Comment by Harsh-Ojha** [comment]
> I had the same issue even though i followed the steps in the video so i researched and used this method.
> 
> 1. Place the favicon.ico in your `public/assets/` folder
> 2. In the layout.tsx file, type this
> 
> ```js
> import { Metadata } from "next";
> 
> export const metadata: Metadata = {
>     title: 'Echo: Speak your mind',
>     description: 'Social media for the people',
>     icons: { icon: '/assets/favicon.ico' },
>   }
> ```
> 
> This worked for me.

Thanks @neophyte-programmer
